benchmark/benchmarks.jl

```diff
@@ -0,0 +1,113 @@
+using BenchmarkTools
+using QuantumOptics
+using OrdinaryDiffEq
+using StochasticDiffEq
+using LinearAlgebra
+using PkgBenchmark
+
+
+const SUITE = BenchmarkGroup()
+
+prob_list = ("schroedinger", "master", "stochastic_schroedinger", "stochastic_master")
+for prob in prob_list
+    SUITE[prob] = BenchmarkGroup([prob])
+    for type in ("pure", "custom")
```

Comment:
May we use "QO types" and "Base array types" instead of pure and custom? These confused me for a second